lib/spack/spack/test/environment.py

```diff
@@ -135,23 +122,15 @@ def test_filter_system_paths(miscellaneous_paths):
     filtered = filter_system_paths(miscellaneous_paths)
     expected = [
         '/usr/local/Cellar/gcc/5.3.0/lib',
+        '/usr/local/lib',
+        '/usr/local/include',
+        '/usr/local/lib64',
```

Comment:
I'm actually confused about this one, let's see if you can shed some light. Previously, `filter_system_paths` used `system_dirs`, which contains things like `['/lib', '/usr/lib64', '/usr/local/include', ...]` with suffixes on them. But it looks like everywhere we used `filter_system_paths`, we were passing it prefixes without any suffixes attached. Am I right? I changed it to use `system_paths` so it should be right now.